src/sentry/models/releasethreshold.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+)
+
+
+class ReleaseThresholdType:
+    TOTAL_ERROR_COUNT = 0
+    NEW_ISSUE_COUNT = 1
+    UNHANDLED_ISSUE_COUNT = 2
+    REGRESSED_ISSUE_COUNT = 3
+    FAILURE_RATE = 4
+    CRASH_FREE_SESSION_RATE = 5
+    CRASH_FREE_USER_RATE = 6
+
+    @classmethod
+    def as_choices(cls):
+        return (
+            (cls.TOTAL_ERROR_COUNT, "Total Error Count"),
+            (cls.NEW_ISSUE_COUNT, "New Issue Count"),
+            (cls.UNHANDLED_ISSUE_COUNT, "Unhandled Issue Count"),
+            (cls.REGRESSED_ISSUE_COUNT, "Regressed Issue Count"),
+            (cls.FAILURE_RATE, "Failure Rate"),
+            (cls.CRASH_FREE_SESSION_RATE, "Crash Free Session Rate"),
+            (cls.CRASH_FREE_USER_RATE, "Crash Free User Rate"),
+        )
+
+
+class ValueType:
+    PERCENT_OVER = 0
+    PERCENT_UNDER = 1
+    ABSOLUTE_OVER = 2
+    ABSOLUTE_UNDER = 3
+
+    @classmethod
+    def as_choices(cls):
+        return (
+            (cls.PERCENT_OVER, "Percent Over"),
+            (cls.PERCENT_UNDER, "Percent Under"),
+            (cls.ABSOLUTE_OVER, "Absolute Over"),
+            (cls.ABSOLUTE_UNDER, "Absolute Under"),
+        )
+
+
+@region_silo_only_model
+class ReleaseThreshold(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    threshold_type = BoundedPositiveIntegerField(choices=ReleaseThresholdType.as_choices())
```

Comment:
hmm just a note for posterity
By keeping keeping these types as an enum + the column an integer, this means we won't be enabling variable threshold types (eg. byo thresholds). If we ever want to add any new threshold types, we'll be required to add them to this enum.

Reply:
Yeah I think that's fine - we can just grow this enum